lib/fluent/plugin/out_rewrite_tag_filter.rb

```diff
@@ -67,13 +69,28 @@ def multi_workers_ready?
 
   def process(tag, es)
     placeholder = get_placeholder(tag)
-    es.each do |time, record|
-      rewrited_tag = rewrite_tag(tag, record, placeholder)
-      if rewrited_tag.nil? || tag == rewrited_tag
-        log.trace("rewrite_tag_filter: tag has not been rewritten", record)
-        next
+    if @emit_mode == :record
```

Comment:
How about preprocessing this in `configure` like below?

```
# in configure
@batch_emit = @emit_mode == :batch

# in process
if @batch_emit
else
end
```

boolean is faster than symbol comprison.

Reply:
Thanks! Fixed in 7a645a5.